bazelisk.py

```diff
@@ -68,6 +68,31 @@
 BAZEL_UPSTREAM = "bazelbuild"
 
 
+def get_env_or_config(name, default=None):
```

Comment:
Ideally we'd only read the actual file once and cache the results.

Reply:
My thought process was that `bazelisk.py` should prioritize simplicity and portability over raw speed (users who want raw speed should be using the native version), but anyway I pushed a second commit to add the memoization.